src/components/filter-modal/filter-modal.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./filter-modal.scss";
+
+import { SearchInput } from "../search-input/search-input";
+
+import ChevronDownIcon from "../../images/icons/chevron-down.svg";
+import ChevronUpIcon from "../../images/icons/chevron-up.svg";
+
+export class FilterModal extends React.Component {
```

Comment:
For consistency, could you please change this to a function component instead of class component, like your previous components ?

Reply:
Yes, will take a look on that rule Monday

Reply:
@surdu check out #134